python/IECoreArnoldTest/RendererTest.py

```diff
@@ -2651,6 +2648,11 @@ def testStatsAndLog( self ) :
 
 			self.assertTrue( "ray counts" in stats )
 
+		# This test runs OK in isolation, but previous uses of ArnoldRenderer seem
+		# to stop Arnold from writing the profile file.
+		if int(arnold.AiGetVersion()[0]) == 7 :
+			return
```

Comment:
We're unlikely to come back and address this if we don't retain the "skipped" message in the test summary as a reminder. In Python 3 we could do this with `self.skipTest()` instead of `return`. For Python 2 compatibility I think we might be able to `raise SkipTest()`.

Reply:
In #4556 it seems like we're already using `self.skipTest` and it seems to be working in python 2 at IE... I get the following once I've added it:

`testStatsAndLog (IECoreArnoldTest.RendererTest.RendererTest) ... skipped 'Profiling broken in Arnold 7'`

Any reason you thought we needed to `raise SkipTest`?

Reply:
Oh, cool. I was just reading the "Added in 3.1" bit of the Python docs : https://docs.python.org/3/library/unittest.html#unittest.TestCase.skipTest. If it's actually there in 2.7 then even better.

Reply:
Addressed in 9ee187dc4a2d0c1ef47a7acdf4280cab60d6f5dd